### ISSUE TITLE
Unable to set default value to the AutoCompleteInput

### ISSUE THREAD

**Issue by icy-meteor** [anubra266/choc-autocomplete#38]
```

  let cats = [
    { id: 'CAT_1', label: 'Kitty 1' },
    { id: 'CAT_2', label: 'Kitty 2'},
  ];

  return
  <AutoComplete openOnFocus suggestWhenEmpty rollNavigation>
    <AutoCompleteInput variant="ghost" bgColor="transparent" />

    <AutoCompleteList fontStyle="normal">
      {cats.map((cat) => (
        <AutoCompleteItem
          key={cat.id}
          value={cat.label}
          textTransform="capitalize"
        >
          {cat.label}
        </AutoCompleteItem>
      ))}
    </AutoCompleteList>
  </AutoComplete>
}

```
Package version "@choc-ui/chakra-autocomplete": "^4.3.3",

I want the default value in the input box to be set to "Kitty 1" whenever the page loads. 
With the above code, the input box is empty on page load which seems like the expected behaviour.

- I tried setting defaultValue to the `<AutoCompleteInput>` component but it doesn't do anything.
- I tried setting value to the <AutoCompleteInput> component. It worked but whenever I focus on the input box first time and click outside the `<AutoCompleteList>`, the value disappears.

What is the right way to go about it?

**Comment by anubra266** [comment]
use The `defaultValues` prop and pass an array.

`<AutoComplete defaultValues={["Kitty 1"]}`

**Comment by icy-meteor** [comment]
I'll try this and get back. Thanks for the quick reply.

**Comment by icy-meteor** [comment]
Okay, now I'm getting an interesting behaviour. Basically reverse of the 2nd point I mentioned.
Now the value seems to get set but the input box shows up empty on page load. When I click on the input box, then the value suddenly shows up. When I click outside now, it continues to stay in the input box.

**Comment by icy-meteor** [comment]
Got it, looks like it also wants me to set  `<AutoCompleteInput value="Kitty 1">` in order to make it show up on page load.
Is this the right behaviour?

**Comment by anubra266** [comment]
Yeah...that would work, but I should handle it internally. I'll work on this, in a moment

**Comment by anubra266** [comment]
Fixed in [v4.4.1](https://github.com/anubra266/choc-autocomplete/releases/tag/v4.4.1)
